activesupport/lib/active_support/callbacks.rb

```diff
@@ -675,10 +675,28 @@ def set_callback(name, *filter_list, &block)
         # <tt>:unless</tt> options may be passed in order to control when the
         # callback is skipped.
         #
-        #   class Writer < Person
-        #      skip_callback :validate, :before, :check_membership, if: -> { age > 18 }
+        #   class Writer < PersonRecord
+        #     attr_accessor :age
+        #     skip_callback :save, :before, :saving_message, if: -> { age > 18 }
         #   end
         #
+        #   writer = Writer.new
+        #   writer.age = 20
+        #   writer.save
+        #
+        # Output:
+        #   - save
+        #   saved
+        #
+        #   young_writer = Writer.new
+        #   young_writer.age = 17
+        #   young_writer.save
```

Comment:
Looks like it is not rendered as intended.

![image](https://user-images.githubusercontent.com/12642/117531400-49956600-b01d-11eb-8067-c49c8c5cc980.png)

Reply:
Oh I didn't notice, thanks. How could I notice it before commit? Running `bundle exec sdoc`?

Reply:
I've run `rdoc; open doc/index.html`.

Reply:
Thanks!